js/app.js

```diff
@@ -1,5 +1,29 @@
-angular.module('GiphyApp', [])
+var app = angular.module('GiphyApp', []);
+
+app.controller('SearchCtrl', ['$scope', '$http', function($scope, $http) {
+	$scope.searchTerm = '';
+	$scope.giphyResult = [];
+
+
+	$scope.$watch('searchTerm', function(newValue, oldValue) {
+		console.log('new value:', newValue);
+		console.log('old value:', oldValue);
+		console.log('.......................');
+		$scope.search();
+	});
+
+	$scope.search = function() {
+		var req = {
+			url: "http://api.giphy.com/v1/gifs/search?q="+$scope.searchTerm+"&limit=100&api_key=dc6zaTOxFJmzC",
+			method: 'GET',
+		};
+
+		$http(req).then(function success(res) {
+			console.log(res);
```

Comment:
Hi Navya, images aren't showing because you aren't pushing your results to ```$scope.giphyResult```. Adding this below line 22 will push each result to your array:
```
			res.data.data.forEach(function(item){
				$scope.giphyResult.push(item);
			});
```